californica/uclalib_californica_restore.yml

```diff
@@ -1,5 +1,27 @@
 ---
 
+- hosts: "californica{{ server_environ }}fedora:californica{{ server_environ }}solr:californica{{ server_environ }}web"
+  become: true
+  become_method: sudo
+  user: ansible
+
+  tasks:
+    - name: Set default restore environment
+      set_fact:
+        restore_environ: "{{ server_environ }}"
+      when: restore_environ is undefined
```

Comment:
This "when" is redundant, as command line arguments (via -e/--extra-vars) override evan playbook defined variables. Left in for clarity.